src/components/carousel/_index.scss

```diff
@@ -6,6 +6,8 @@
 
 	display: grid;
 	grid-template-areas: "controls" "carousel" "indicators";
+	grid-template-columns: 100%;
+	grid-template-rows: min-content;
```

Comment:
@nschubach It's working just with `grid-template-columns`. Is there any specific reason for adding `grid-template-rows`

Reply:
I think that would be necessary for vertical languages?  It's purely a guess right now since I didn't have time to test with `writing-mode: vertical-lr;` or anything like that.

Reply:
Can you add a comment on that line, in case if anyone removes

Reply:
```suggestion
```

Reply:
Removed it since the columns will be different for writing modes: https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_grid_layout/Grids_logical_values_and_writing_modes#logical_values_for_alignment